src/main/java/de/tarent/resources/CoursesResource.java

```diff
@@ -19,7 +19,7 @@
 @Transactional
 public class CoursesResource {
 
-    private static final Response NOT_FOUND_RESPONSE = Response.status(NOT_FOUND).build();
+    static final Response NOT_FOUND_RESPONSE = Response.status(NOT_FOUND).build();
```

Comment:
package private ist a bad form
you have to at least write `/*package*/ static final`
better to just make it public

Reply:
Why is package private bad if one wants to make classes/methods etc. accessible only inside the same package (here resources)?

Reply:
It is by now an Antipattern. If you want to restrict access to implementation details modules are the way to go.